src/app/core/layout/main-layout/main-layout.component.html

```diff
@@ -26,9 +26,8 @@
     <button
       mat-flat-button
       color="primary"
-      routerLink="/tickets/board"
+      routerLink="https://www.youtube.com/watch?v=B6en-O5yF0o"
```

Comment:
To make it work this has to be modified a little bit:
` 
<a
    mat-flat-button
    color="primary"
    href="http://google.com"
  >
    <mat-icon>dashboard</mat-icon>
    Board</a>
`
`routerLink` is only used within the Angular app context and it would result in 404 page not found.

Reply:
I thought that could be the case here, thanks!